ISPN-12657 Cross-Site automatic state transfer (#9202)
infinispan/infinispan: documentation/src/main/asciidoc/topics/proc_cli_xsite_state_transfer_mode.adoc

[id='cli_xsite_push-{context}']

> = Set the Cross Site state transfer mode

Reviewer: ```suggestion
= Configuring Cross-Site State Transfer Modes
```

> Enable or disable the automatic cross site state transfer.

Reviewer: ```suggestion
You can configure cross-site state transfer operations to happen automatically when {brandname} detects that backup locations come online or manually, which is the default.
```

.Procedure

. Create a CLI connection to {brandname}.

> . Use the [command]`site` command to retrieve or set the state transfer mode, as in the following example:

Reviewer: ```suggestion
. Use the [command]`site` command to configure state transfer modes, as in the following example:
```

+
----

> # retrives the current state transfer mode

Reviewer: ```suggestion
# Retrieve the current state transfer mode
```

//containers/default]> site state-transfer-mode --cache=cacheName --site=NYC
"MANUAL"

> # sets the state transfer mode

Reviewer: ```suggestion
# Configure automatic state transfer operations for a cache and backup location.
```
